node/node.go

```diff
@@ -395,7 +395,15 @@ func (n *Node) ValidateBatch(ctx context.Context, header *core.BatchHeader, blob
 	}
 
 	pool := workerpool.New(n.Config.NumBatchValidators)
-	return n.Validator.ValidateBatch(header, blobs, operatorState, pool)
+	err = n.Validator.ValidateBatch(header, blobs, operatorState, pool)
+	if err != nil {
+		h, hashErr := operatorState.Hash()
```

Comment:
How to use this hash returned from node?

Reply:
This only gets embedded in the error that gets returned. 
Batcher also logs the hash so that it can be manually compared for debugging